src/rules.rs

```diff
@@ -748,18 +748,10 @@ pub(crate) fn indentation() -> IndentDsl {
                 , baz
                 }: foo
             "#)
-        
+
         .rule("Indent lambda body")
             .inside(p(NODE_LAMBDA) & p(not_on_top_level) & p(pattern_not_newline))
             .set(Indent)
-        .rule("Indent newline lambda body")
-            .inside(p(NODE_LAMBDA) & p(not_on_top_level) & p(pattern_newline) & p(lambda_inside_node_pattern))
-            .not_matching(p(TOKEN_COMMENT)) 
-            .set(Indent)
-         .rule("Indent newline lambda body")
-            .inside(p(NODE_LAMBDA) & p(not_on_top_level) & p(pattern_newline) & p(lambda_outside_node_pattern))
-            .not_matching(p(TOKEN_COMMENT) | p(VALUES)) 
-            .set(Indent)
```

Comment:
Haven't check other rules, but these two seem unnecessary with the change to the engine.

Reply:
Beside the `explain_smoke_test` failure`, I found two other rules that can be removed which are regarding node binops and inline node if_else.